server/docker-entrypoint.sh

```diff
@@ -25,5 +25,10 @@ else
     echo "[KEYCLOAK DOCKER IMAGE] Using the embedded H2 database"
 fi
 
+if [ "$JGROUPS_SETUP" != "" ]; then
+    echo "[KEYCLOAK DOCKER IMAGE] Using custom JGroups setup $JGROUPS_SETUP"
+    /bin/sh /opt/jboss/keycloak/bin/change-jgroups.sh $JGROUPS_SETUP
```

Comment:
This won't forward OS signals: I gave it a try and didn't configure the DNS properly, and the Docker container got stuck in the "Looking up initial hosts for DNS name" loop. It couldn't be stopped with a SIGTERM, since bash doesn't forward signals to child processes.

I exchanged this line with the following lines and was able to stop the Docker image properly when stuck in the initial discovery phase.

```bash
trap 'kill -TERM $CHILD_PID; exit 1' TERM INT
/opt/jboss/keycloak/bin/change-jgroups.sh "$JGROUPS_SETUP" &
CHILD_PID=$!
wait $CHILD_PID
trap - TERM INT
wait $CHILD_PID
```

Reply:
Good point. I think I just copied this from how `change-database.sh` is invoked. I'd suggest a switch to `/bin/bash`, but I would prioritize similarity with change-database.